gtk-3.0/assets/checkbox-checked-dark.svg

```diff
@@ -2,7 +2,9 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="133pt" height="133pt" viewBox="0 0 133 133" version="1.1">
 <g id="surface1">
 <path style=" stroke:none;fill-rule:nonzero;fill:#%TXT_BG%;fill-opacity:1;" d="M 16.667969 8.332031 L 116.667969 8.332031 C 121.257812 8.332031 125 12.078125 125 16.667969 L 125 116.667969 C 125 121.257812 121.257812 125 116.667969 125 L 16.667969 125 C 12.078125 125 8.332031 121.257812 8.332031 116.667969 L 8.332031 16.667969 C 8.332031 12.078125 12.078125 8.332031 16.667969 8.332031 Z M 16.667969 8.332031 "/>
-<path style=" stroke:none;fill-rule:nonzero;fill:#%FG%;fill-opacity:1;" d="M 16.667969 0 C 7.421875 0 0 7.421875 0 16.667969 L 0 116.667969 C 0 125.910156 7.421875 133.332031 16.667969 133.332031 L 116.667969 133.332031 C 125.910156 133.332031 133.332031 125.910156 133.332031 116.667969 L 133.332031 16.667969 C 133.332031 7.421875 125.910156 0 116.667969 0 Z M 16.667969 8.332031 L 116.667969 8.332031 C 121.289062 8.332031 125 12.042969 125 16.667969 L 125 116.667969 C 125 121.289062 121.289062 125 116.667969 125 L 16.667969 125 C 12.042969 125 8.332031 121.289062 8.332031 116.667969 L 8.332031 16.667969 C 8.332031 12.042969 12.042969 8.332031 16.667969 8.332031 Z M 16.667969 8.332031 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:#%BG%;fill-opacity:1;" d="M 16.667969 0 C 7.421875 0 0 7.421875 0 16.667969 L 0 116.667969 C 0 125.910156 7.421875 133.332031 16.667969 133.332031 L 116.667969 133.332031 C 125.910156 133.332031 133.332031 125.910156 133.332031 116.667969 L 133.332031 16.667969 C 133.332031 7.421875 125.910156 0 116.667969 0 Z M 16.667969 8.332031 L 116.667969 8.332031 C 121.289062 8.332031 125 12.042969 125 16.667969 L 125 116.667969 C 125 121.289062 121.289062 125 116.667969 125 L 16.667969 125 C 12.042969 125 8.332031 121.289062 8.332031 116.667969 L 8.332031 16.667969 C 8.332031 12.042969 12.042969 8.332031 16.667969 8.332031 Z M 16.667969 8.332031 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:#000000;fill-opacity:0.20;" d="M 16.667969 0 C 7.421875 0 0 7.421875 0 16.667969 L 0 116.667969 C 0 125.910156 7.421875 133.332031 16.667969 133.332031 L 116.667969 133.332031 C 125.910156 133.332031 133.332031 125.910156 133.332031 116.667969 L 133.332031 16.667969 C 133.332031 7.421875 125.910156 0 116.667969 0 Z M 16.667969 8.332031 L 116.667969 8.332031 C 121.289062 8.332031 125 12.042969 125 16.667969 L 125 116.667969 C 125 121.289062 121.289062 125 116.667969 125 L 16.667969 125 C 12.042969 125 8.332031 121.289062 8.332031 116.667969 L 8.332031 16.667969 C 8.332031 12.042969 12.042969 8.332031 16.667969 8.332031 Z M 16.667969 8.332031 "/>
```

Comment:
Sure, using FG is okay for light themes, but even then it'll still look weird, because most widgets create their border by darkening the BG color.

Huh? Layering a transparent black onto something will never lighten it, it will always darken it. Sure, on dark themes it will have less effect, but that's consistent with the behavior of widget borders on dark themes anyways. Those borders are just shade(0.80, color); this is basically the same thing.